package.json

```diff
@@ -25,6 +27,8 @@
   },
   "devDependencies": {
     "bl": ">=0.9.0 <0.10.0-0",
+    "coveralls": "^2.11.2",
```

Comment:
Hey @rvagg I'll update this shortly. @dcousens, one could argue the other side that you're more likely to get bugs with `^`, since you're pulling in new features (in a perfect world, this would not break the API but with new features comes more code churn). I tend towards `^`, because I like to live dangerously, but I can understand the other side of the argument.